deny.toml

```diff
@@ -90,7 +90,12 @@ ignore = [
 
     # Very low-severity TTY error we will never hit: code that's not compiled
     # into production, only on windows, and only with a custom allocator.
-    "RUSTSEC-2021-0145"
+    "RUSTSEC-2021-0145",
+
+    # This ignores an error due to the now unmaintained derivative crate. This
+    # crate is a dependency for some of the ark crates, so we need to ignore the
+    # advisory.
+    "RUSTSEC-2024-0388"
```

Comment:
@graydon @leighmcculloch I don't think we have another option, but wanted to bring your attention so this. The derivative crate was removed from the latest ark crates, but we aren't using those. We should probably upgrade (if possible) for the next protocol upgrade (cc @jayz22).

Reply:
yeah, totally not pressing (there's no bug in `derivative` it's just unmaintained). I would wait until our next upgrade to the ark crates. we haven't done across-the-board bumps to all soroban dependencies yet, but now that we have a [more palatable versioning story on soroban](https://github.com/stellar/stellar-core/pull/4514) I think it'd be good to start doing on protocol boundaries.